testing/startgomodfs/startgomodfs.go

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+// The startgomodfs binary is used in CI tests to start a gomodfs server on
+// Windows, because Powershell-in-YAML-in-Github-Actions with shell quoting
+// is hard. But then for consistency it also does Linux & macOS, even though
+// those are trivial from YAML.
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net"
+	"net/http"
+	"os"
+	"os/exec"
+	"reflect"
+	"runtime"
+	"syscall"
+	"time"
+)
+
+const (
+	binPath = "./gomodfs.exe" // we use '.exe' in CI even on non-Windows; why not?
```

Comment:
As an alternative to the go env comment above, it's maybe a bit less confusing and simpler to just use `go run`?